packages/components/src/components/date-picker/date-picker-native.jsx

```diff
@@ -20,6 +20,8 @@ const Native = ({
     value,
 }) => {
     const [is_focused, setIsFocused] = React.useState(0);
+    const [old_value, setOldValue] = React.useState('');
```

Comment:
Can we maybe call it `previous_value` instead if that's what it refers to?

Reply:
Will update thanks.